packages/components/src/palette-edit/test/index.tsx

```diff
@@ -16,9 +16,9 @@ async function clearInput( input: HTMLInputElement ) {
 	await click( input );
 
 	// Press backspace as many times as the input's current value
-	for ( const _ of Array( input.value.length ) ) {
-		await press.Backspace();
-	}
+	[ ...Array( input.value.length ) ].forEach(
+		async () => await press.Backspace()
+	);
```

Comment:
Let's avoid this change and instead configure the rule to allow `_`-prefixed variables.

We don't want to be rewriting valid code to appease the linter. Please revert this change.

Reply:
The problem is not, that the `_` variables are not allowed, the problem is that the `_` is not used in the `for`-block. So that would mean, that I need to add a `eslint-disable` here. Are you agreeing with that?

Reply:
Let's revert this and add the appropriate disable comment.

Reply:
As discussed, reverted the change here https://github.com/WordPress/gutenberg/pull/62925/commits/2b21b06932ab2f53944da3fed7b7df9eed0504b5 and added "eslint-disable".